pkg/util/keyvault/keyvault.go

```diff
@@ -196,7 +196,7 @@ func (m *manager) GetCertificateSecret(ctx context.Context, secretName string) (
 	return key, certs, nil
 }
 
-func (m *manager) GetSecret(ctx context.Context, secretName string) (azkeyvault.SecretBundle, error) {
+func (m *manager) GetSecret(ctx context.Context, secretName string, version string) (azkeyvault.SecretBundle, error) {
```

Comment:
Will `version` be used at a later time? why add this now?

Reply:
@SrinivasAtmakuri it seems this isn't used?

Reply:
should line 200 be `m.kv.GetSecret(ctx, m.keyvaultURI, secretName, version)`?

Reply:
Made the change.